### PR TITLE
doc: remove redundant word

### PR DESCRIPTION
**Pull request by iliul**
# Describe what this PR does #

remove redundant text "the"

### CODE REVIEW

**Review by Yuggupta27 — CHANGES_REQUESTED** [review]
Thanks for the PR @iliul ! Just some minor nit.

**Comment by Yuggupta27** [comment]
Also, U can consider removing the additional text in the PR description if you want :)

File: examples/csi-config-map-sample.yaml [hunk 1]
```diff
@@ -13,8 +13,8 @@ kind: ConfigMap
 # To add more clusters or edit MON addresses in an existing configmap, use
 # the `kubectl replace` command.
 # The <rados-namespace> is optional and represents a radosNamespace in the pool.
-# If any given all the of the rbd images, snapshots and other metadata
-# will be store within the radosNamespace.
+# If any given, all of the rbd images, snapshots, and other metadata will be stored
+# within the radosNamespace.
```
Comment (nixpanic) [inline examples/csi-config-map-sample.yaml:17]: This now fails `yamlllint` with the following error:
```
 ./examples/csi-config-map-sample.yaml
  16:81     error    line too long (83 > 80 characters)  (line-length)
```

You can run `make containerized-test` to run the verification locally.

**Review by nixpanic — CHANGES_REQUESTED** [review]

**Review by nixpanic — DISMISSED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

**Pull request merged**